package.json

```diff
@@ -50,5 +51,12 @@
     "@types/hash-sum": "^1.0.0",
     "@types/globby": "^9.1.0",
     "@types/escape-html": "^0.0.20"
+  },
+  "dependencies": {
```

Comment:
Since VuePress is a monorepo, the `postinstall ` setup should be put in `packages/vuepress/package.json`

Reply:
changed